### PR TITLE
Prepare 1.1.0-RC

### PR DESCRIPTION
**Pull request by sandwwraith**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+Using 1.1.0-RC, you can mark inline classes as `@Serializable` and use them in other serializable classes. 
+Unsigned integer types (`UByte`, `UShort`, `UInt` and `ULong`) are serializable as well and have special support in JSON.
+This feature requires Kotlin compiler 1.4.30-RC and enabling new IR compilers for for [JS](https://kotlinlang.org/docs/reference/js-ir-compiler.html) and [JVM](https://kotlinlang.org/docs/reference/whatsnew14.html#new-jvm-ir-backend).
+
+You can learn more in the [documentation](docs/inline-classes.md)
+and corresponding [pull request](https://github.com/Kotlin/kotlinx.serialization/pull/1244).
+
+### Other features
+
+  * Add serializerOrNull function for KType and Type arguments (#1164)
```
Comment (qwwdfsad) [inline CHANGELOG.md:25]: Here and below I suggest wrapping all methods/classes into backticks, so they are properly rendered

**Review by qwwdfsad — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+Due to changes in calling conventions between compiler plugin and serialization core runtime, this release requires 
+Kotlin version at least 1.4.30-M1. However, this changes should not affect your code,
+because only deprecated functions were removed from public API.
+See [corresponding PR](https://github.com/Kotlin/kotlinx.serialization/pull/1260) for the details.
+
+### Experimental support for inline classes (IR only)
+
+Using 1.1.0-RC, you can mark inline classes as `@Serializable` and use them in other serializable classes. 
+Unsigned integer types (`UByte`, `UShort`, `UInt` and `ULong`) are serializable as well and have special support in JSON.
+This feature requires Kotlin compiler 1.4.30-RC and enabling new IR compilers for for [JS](https://kotlinlang.org/docs/reference/js-ir-compiler.html) and [JVM](https://kotlinlang.org/docs/reference/whatsnew14.html#new-jvm-ir-backend).
```
Comment (Him188) [inline CHANGELOG.md:18]: Maybe duplicated 'for'?
```suggestion
This feature requires Kotlin compiler 1.4.30-RC and enabling new IR compilers for [JS](https://kotlinlang.org/docs/reference/js-ir-compiler.html) and [JVM](https://kotlinlang.org/docs/reference/whatsnew14.html#new-jvm-ir-backend).
```

**Review by Him188 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (sandwwraith) [reply CHANGELOG.md:18]: Indeed, thanks!

**Comment by Whathecode** [comment]
The new Kotlin [already release mentions this version](https://kotlinlang.org/releases.html). :) I guess a new release is imminent! 🥳

**Pull request merged**